interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -709,10 +705,14 @@ function ViewModel() {
                 // Do the next one
                 self.addNZBFromFile(files, fileindex)
             } else {
-                // Hide notification
-                hideNotification('.main-notification-box-uploading')
                 // Refresh
                 self.refresh();
+                // Hide notification
+                hideNotification('.main-notification-box-uploading')
+                // Reset the form
+                $('#modal-add-nzb form').trigger('reset');
+                $('#nzbname').val('')
```

Comment:
you have trailing `;` above but dont do it on other lines..  cosmetic but could lead to issues if not explicitly defined when making code changes later on.